### PR TITLE
BrainStorm: Behaviour, SetEntityLookTargetRandomly

### PR DESCRIPTION
**Pull request by mrsterner**
I added some basic mappings for when an entity is selecting a look target at a chance.
This is for a Task in the Brain part of minecraft AI

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	METHOD m_uabosmqw (Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;Ljava/util/function/Predicate;FLnet/minecraft/unmapped/C_lygsomtd$C_gvxfhlod;Lnet/minecraft/unmapped/C_ujlmiamh;Lnet/minecraft/unmapped/C_ujlmiamh;)Lnet/minecraft/unmapped/C_smovfozc;
+		ARG 4 memory
+		ARG 5 memory2
+	METHOD m_xpopbuma of (FLnet/minecraft/unmapped/C_hvilmtvi;Ljava/util/function/Predicate;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 distance
+		ARG 1 range
+		ARG 2 predicate
+	CLASS C_gvxfhlod Ticker
+		FIELD f_efesnins countdownToStart I
+		FIELD f_rdvkztgg intProvider Lnet/minecraft/unmapped/C_hvilmtvi;
```
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping:28]: is there not a better name for this? its purpose isn't described

**Review by eristhea — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping [hunk 2]
```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/unmapped/C_lygsomtd net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly
+	METHOD m_aecmvuxy of (Lnet/minecraft/unmapped/C_ogavsvbr;FLnet/minecraft/unmapped/C_hvilmtvi;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 entityType
+		ARG 1 chance
+		ARG 2 range
+	METHOD m_ipziivps (Lnet/minecraft/unmapped/C_usxaxydn;FLnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 2 targetLivingEntity
+	METHOD m_nozcjkxz (Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;Lnet/minecraft/unmapped/C_ujlmiamh;Ljava/util/function/Predicate;FLnet/minecraft/unmapped/C_lygsomtd$C_gvxfhlod;Lnet/minecraft/unmapped/C_ujlmiamh;Lnet/minecraft/unmapped/C_bdwnwhiu;Lnet/minecraft/unmapped/C_usxaxydn;J)Z
+		ARG 7 livingEntiity
```
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping:9]:
```suggestion
		ARG 7 livingEntity
```

File: mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	METHOD m_nteqkjjv (Lnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 0 predicate
+	METHOD m_ouvkkubu of (FLnet/minecraft/unmapped/C_hvilmtvi;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 chance
+		ARG 1 range
+	METHOD m_shrczqhx (Lnet/minecraft/unmapped/C_ogavsvbr;Lnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 1 entity
+	METHOD m_swcvkveo (Ljava/util/function/Predicate;FLnet/minecraft/unmapped/C_lygsomtd$C_gvxfhlod;Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;)Lcom/mojang/datafixers/kinds/App;
+		ARG 3 behaviour
```
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping:18]:
```suggestion
		ARG 3 behavior
```
britishism, I need to poke the dictionary about this one it shouldn't been allowed

File: mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	METHOD m_uabosmqw (Lnet/minecraft/unmapped/C_rcqaryar$C_loqvwuht;Ljava/util/function/Predicate;FLnet/minecraft/unmapped/C_lygsomtd$C_gvxfhlod;Lnet/minecraft/unmapped/C_ujlmiamh;Lnet/minecraft/unmapped/C_ujlmiamh;)Lnet/minecraft/unmapped/C_smovfozc;
+		ARG 4 memory
+		ARG 5 memory2
```
Comment (eristhea) [inline mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping:21]: the function of this parameter is not described!

**Review by eristhea — COMMENTED** [review]
a few more comments for you

File: mappings/net/minecraft/entity/ai/brain/Behaviour.mapping [hunk 5]
```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/unmapped/C_rcqaryar net/minecraft/entity/ai/brain/Behaviour
```
Comment (EnnuiL) [inline mappings/net/minecraft/entity/ai/brain/Behaviour.mapping:1]: oh dear this is quite british

**Review by EnnuiL — CHANGES_REQUESTED** [review]
proper review will come later

File: mappings/net/minecraft/entity/ai/brain/Behaviour.mapping (hunk 5 above)
Comment (ix0rai) [reply mappings/net/minecraft/entity/ai/brain/Behaviour.mapping:1]: didn't even notice that was the entire class wow

File: mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping [hunk 6]
```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/unmapped/C_lygsomtd net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping:1]:
```suggestion
CLASS net/minecraft/unmapped/C_lygsomtd net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomlyTask
```
Should this have the `Task` suffix to be consistent with other classes in this package?

Also I don't know if there's a convention on where adverbs that modify the verb of task class names go, but it should probably be brought up (`SetEntityLookTargetRandomly` vs `RandomlySetEntityLookTarget`).

File: mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+	METHOD m_aecmvuxy of (Lnet/minecraft/unmapped/C_ogavsvbr;FLnet/minecraft/unmapped/C_hvilmtvi;)Lnet/minecraft/unmapped/C_mdnathub;
+		ARG 0 entityType
+		ARG 1 chance
+		ARG 2 range
+	METHOD m_ipziivps (Lnet/minecraft/unmapped/C_usxaxydn;FLnet/minecraft/unmapped/C_usxaxydn;)Z
+		ARG 2 targetLivingEntity
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/ai/brain/task/SetEntityLookTargetRandomly.mapping:7]:
```suggestion
		ARG 2 targetedLivingEntity
```
iiuc this should be a noun phrase and not a verb phrase

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Comment by eristhea** [comment]
@mrsterner given that this PR hasn't seen action from you in 4 months, I'm closing it. feel free to reopen and tag me (@ix0rai) if you come back to it.

**Pull request closed without merging**